client/app/application.coffee

```diff
@@ -47,34 +46,49 @@ class Application extends Mn.Application
 
         # render components when app starts
         @on 'start', ->
+            @layout.render()
+
             @contacts.fetch reset: true
             @tags.fetch reset: true
 
-            @layout.render()
-
             # prohibit pushState because URIs mapped from cozy-home rely on
             # fragment
             Backbone.history.start pushState: false if Backbone.history
 
 
+    getSelected: ->
+        view = @layout.getChildView 'content'
+        view.children.reduce (memo, view) ->
```

Comment:
> I find using a reduce for this case less readable

I though it's a perfect use case for reduce, but I ask myself if a `filter().map()` chain won't be better… I'm definitely not fan of `for` loop as long as we try to filter the list (and not explicitely iterate over it). Any suggestion?